Bottom: c3701c5eb950aca53190db6d852de0895795faa0
Top:    04ade1c429d0971a1ce7e76121a4a911b8849a81
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Move mariadb config to maria role


---
diff --git a/roles/docker-common/tasks/main.yml b/roles/docker-common/tasks/main.yml
index bc8aac33..05894452 100644
--- a/roles/docker-common/tasks/main.yml
+++ b/roles/docker-common/tasks/main.yml
@@ -6,7 +6,6 @@
 
 - import_tasks: packages.yml
 - import_tasks: sshd_settings.yml
-- import_tasks: mariadb_settings.yml
 
 - import_tasks: configure.yml
 
diff --git a/roles/docker-common/tasks/mariadb_settings.yml b/roles/docker-common/tasks/mariadb_settings.yml
deleted file mode 100644
index 54508dd3..00000000
--- a/roles/docker-common/tasks/mariadb_settings.yml
+++ /dev/null
@@ -1,15 +0,0 @@
----
-# file: roles/docker-common/tasks/mariadb_settings.yml
-
-- name: Set host ssh ListenAddress
-  template:
-    src: mariadb-bind-address.j2
-    dest: /etc/my.cnf.d/mariadb-bind-address.cnf
-  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
-
-- name: Restart MariaDB service
-  service:
-    name: mariadb
-    enabled: yes
-    state: restarted
-  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
diff --git a/roles/mariadb/tasks/main.yml b/roles/mariadb/tasks/main.yml
index b27a0af7..70e29ec6 100644
--- a/roles/mariadb/tasks/main.yml
+++ b/roles/mariadb/tasks/main.yml
@@ -42,3 +42,15 @@
     permanent: true
     state: enabled
   when: mariadb_open_firewall
+- name: Set host ssh ListenAddress
+  template:
+    src: mariadb-bind-address.j2
+    dest: /etc/my.cnf.d/mariadb-bind-address.cnf
+  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
+
+- name: Restart MariaDB service
+  service:
+    name: mariadb
+    enabled: yes
+    state: restarted
+  when: ansible_br0 is defined and ansible_br0.ipv4 is defined and ansible_br0.ipv4.address is defined
diff --git a/roles/docker-common/templates/mariadb-bind-address.j2 b/roles/mariadb/templates/mariadb-bind-address.j2
similarity index 100%
rename from roles/docker-common/templates/mariadb-bind-address.j2
rename to roles/mariadb/templates/mariadb-bind-address.j2
